examples/aggregator/proto/aggregator.proto

```diff
@@ -20,11 +20,18 @@ package oak.examples.aggregator;
 
 import "google/protobuf/empty.proto";
 
-message Vector {
-  repeated uint64 items = 1;
+message SerializedSparseVector {
+  repeated uint32 indices = 1;
+  repeated float values = 2;
+}
+
+// Message sent by clients.
+message Sample {
+  // Data with different Bucket IDs should not be mixed (aggregated).
```

Comment:
Clarify this comment, it is not clear whether it is asking clients to do something (not sure what), or whether it is describing a guarantee offered by the server.